lib/components/narrative/default/default-itinerary.js

```diff
@@ -193,6 +196,18 @@ class DefaultItinerary extends NarrativeItinerary {
     const isFlexItinerary = itinerary.legs.some(isFlex)
     const isCallAhead = itinerary.legs.some(isReservationRequired)
     const isContDropoff = itinerary.legs.some(isContinuousDropoff)
+    const { SvgIcon } = this.context
+
+    Object.keys(accessibilityScoreGradationMap || {}).forEach((key) => {
```

Comment:
We're already looping over this on line 319. Is there any reason this code can't be run inside that loop?